readme.md

```diff
@@ -10,7 +10,7 @@
 $ yarn add @shelf/jest-dynamodb --dev
 ```
 
-Make sure `aws-sdk` is installed as a peer dependency. And `java` runtime available for running [DynamoDBLocal.jar](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html)
+Make sure 3 `@aws-sdk` packages is installed as a peer dependency. And `java` runtime available for running [DynamoDBLocal.jar](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html)
```

Comment:
I think we don't need peer deps here, as `jest-dynamodb` is always a devDep.
Don't you think so?

Reply:
I think we don't need it, sorry for misinforming you.